Jellyfin.Server.Implementations/Item/PeopleRepository.cs

```diff
@@ -61,7 +61,7 @@ public void UpdatePeople(Guid itemId, IReadOnlyList<PersonInfo> people)
         using var transaction = context.Database.BeginTransaction();
 
         context.PeopleBaseItemMap.Where(e => e.ItemId == itemId).ExecuteDelete();
-        foreach (var item in people)
+        foreach (var item in people.DistinctBy(e => e.Id)) // yes for __SOME__ reason there can be duplicates.
```

Comment:
This comment should be prefixed by TODO, so it can be investigated/reported upstream or whatever action needs to be taken later. Also, above the foreach for readability.